data/config/packages.json

```diff
@@ -1456,8 +1451,14 @@
 				"mangohud"
 			],
 			"repos": {
-				"all": [
-					"ppa:flexiondotorg/mangohud"
```

Comment:
So here we should have a "focal": [
					"ppa:flexiondotorg/mangohud"
				],

Type statement so that it works for that series.  Thanks for adding the hirsute, impish and jammy universe entries - thats good news.

Reply:
sounds right ... adding this